test/integration/fixtures/next-package-not-resolvable-in-base-dir/clear-node-modules-after-functions-bundling-plugin/index.js

```diff
@@ -0,0 +1,12 @@
+const fs = require(`fs`)
+const path = require(`path`)
+
+module.exports = {
+  onPostBuild: async () => {
+    const movedDir = path.join(process.cwd(), `next-app`, `node_modules2`)
```

Comment:
Ah sorry - this is not pulled in test - it's just new one that uses `netlify serve` and this "build plugin" is meant to get rid of `node_modules` after functions were bundled but before they started to be served. However when I just deleted `node_modules` it was problematic, sometimes getting ENOENT errors so instead of deleting it I just move `node_modules` to `node_modules2` so it no longer is used. The reason to check if `node_modules2` exist and deleting it if it does is because running the same test twice in a row would crash otherwise.

And the reason to remove `node_modules` are just functions bundling quirks I encountered:

I did use absolute path to `next-server` (so something like `/Users/misiek/dev/repo/next-app/node_modules/next/<path-to-next-server>`) when generating lambda handler at some point and it worked locally, but not when deployed, because functions bundler doesn't seem to handle absolute paths correctly and left those imports as they are instead of bundling it. Things appeared to be working locally because those `node_modules` still existed.